cilium-health/launch/endpoint.go

```diff
@@ -228,7 +228,7 @@ func LaunchAsEndpoint(owner endpoint.Owner, hostAddressing *models.NodeAddressin
 	ep.SetDefaultOpts(option.Config.Opts)
 
 	// Give the endpoint a security identity
-	ep.UpdateLabels(owner, labels.LabelHealth, nil)
+	ep.UpdateLabels(owner, labels.LabelHealth, nil, true)
```

Comment:
Do we need to do this in the health case? Shouldn't the identity for health always be allocated locally? I guess it can't hurt to do this, was just wondering.

Reply:
You are right that the health identity will be local and not be async anyway but it seems correct to enforce blocking mode. We do not want to health traffic with init identity on the wire if the implementation of `UpdateLabels()` ever changes.

Reply:
Makes sense.